core/build.gradle

```diff
@@ -86,7 +86,8 @@ dependencies {
         playApi "com.google.android.gms:play-services-cast:$playServicesVersion"
         api "com.google.android.support:wearable:$wearableSupportVersion"
     } else {
-        System.out.println("core: free build hack, skipping some dependencies")
+        System.out.println("core: free build hack, skipping some dependencies and bundling conscrypt ("+"$conscryptVersion"+")")
```

Comment:
I think the variable can be inlined without adding `"+"`

Reply:
> I think the variable can be inlined without adding `"+"`

I think you're correct. I tried adding the variable inside the text string and it got substituted correctly. I'll change it later (I also see there's a conflict in build.gradle that I need to adjust).